app/main.py

```diff
@@ -2,0 +1,24 @@
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError as e:
+            print(e)
+            return "All friends should be vaccinated"
+        except NotWearingMaskError as e:
+            masks_to_buy += 1
+            print(e)
+
+    if masks_to_buy:
+        return f"Friends should buy {masks_to_buy} masks"
```

Comment:
Ensure the message is clear and informative. You might want to specify that the friends need to buy masks before they can enter the cafe.